src/data/polynomial/derivative.lean

```diff
@@ -0,0 +1,248 @@
+/-
+Copyright (c) 2018 Chris Hughes. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Chris Hughes, Johannes Hölzl, Scott Morrison, Jens Wagemaker
+-/
```

Comment:
This file deserves an updated authorship line. I think @shingtaklam1324 did this? If that's correct, we should put their name here. 

(You can check with git blame, I guess.)

Reply:
No, if I recall correctly, Shing worked on partial derivatives of multivariate polynomials.

Reply:
Johan is correct. I did not write `polynomial.derivative`, I did `mv_polynomial.pderivative`. I'm fairly sure Chris did `polynomial.derivative`.